main.swift

```diff
@@ -361,9 +361,22 @@ do {
     }
     
     try performCommand(description: "Copying template folder") {
+        let ignorableItems = ["readme.md", "license"]
```

Comment:
How about using a `Set` here instead, to improve lookup time for `contains()`. Minor detail I guess 🙂